program/Makefile

```diff
@@ -1,2 +1,21 @@
 PDFGrader.exe: src/main/System.vala src/main/QuestionSet.vala src/main/Question.vala src/main/Mark.vala src/main/Save.vala
 	valac -o PDFGrader.exe --pkg=gtk+-3.0 --pkg=poppler-glib --pkg cairo --pkg gee-0.8 --pkg glib-2.0 --pkg libxml-2.0 --pkg posix src/main/System.vala src/main/QuestionSet.vala src/main/Question.vala src/main/Mark.vala src/main/Save.vala src/main/Lock.vala src/main/Export.vala
+
+clean: 
+	rm -f Makefile.in
```

Comment:
Looks like you missed the *.c and *.o files. Also I would see what you can do to get autotools to clean up after itself. if you aren't able to get it working, I would suggest adding another target like 'clean-autotools' and remove all the generated autotools files minus the .tar and .exe

Reply:
And use it at the end of maintain.sh